kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -558,6 +567,8 @@
         set(this.dragActive, true);
       },
       openSelectResources(section_id) {
+        //initialize workingResourcePool
+        this.initializeWorkingResourcePool();
```

Comment:
Is there a reason this couldn't happen within the ResourceSelection component instead?